services/app/src/components/gameStatus/GameInfo.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import './gameStatus.css';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import CheckMark from './statusComponents/checkmark.svg';
+import SkippedIcon from './statusComponents/skippedIcon.svg';
+import BulletDot from './statusComponents/bulletDot.svg';
+
+const GameInfo = ({ userCount }) => {
+  const playersDone = 50;
+  const playersSkipped = 10;
+
+  return (
+    <div className="game-info-row">
+      <div className="game-info-status player-count">
+        <p>
+          {userCount === 1
```

Comment:
As per other PR (https://github.com/ZenitechSoftware/planning-chess/pull/59/files) `user` was changed to `player`. Please refactor variable from userCount to playerCount